チャンネルを追加しようとすると「Slackになかなか接続できません」と表示される

添付の通り、SlackのConfigurator for DeepL Translatorのワークフローを使い、
チャンネルを追加しようとすると「Slackになかなか接続できません」と表示されます。

- アプリケーションはデプロイされている
- 既存のチャンネルの設定のままであれば、ワークフローでConfirmを押下しても成功する
- ` slack trigger create --trigger-def triggers/configurator.ts` で新しくリンク作成しても事象は変わらない
ことは確認しました。

数ヶ月前は同様の事象は確認されませんでしたが、日を跨いでも発生します。

こちらはやはり、Slack側の通信が原因となるでしょうか。

![スクリーンショット 2023-09-13 12 04 53](https://github.com/slack-samples/deno-message-translator/assets/61672244/002ec9d9-40fd-4ebd-bfcc-387d7531f8ff)

@seratch 
ご返信いただきありがとうございます！

`Error: Failed to update a trigger! (response: {"ok":false,"error":"invalid_arguments","response_metadata":{"messages":["[ERROR] no more than 20 items allowed [json-pointer:/event/channel_ids]"]}})`

ログを確認してみたところ、上記のエラーが表示されていたため
登録チャンネル数の上限値に達したと思われます。
こちらはアプリ側で上限数を増やすことや、そうでない場合仕様変更が可能となるものなのでしょうか。
ご検討いただければ幸いです🙇‍♂️

--- a/functions/configure.ts
+++ b/functions/configure.ts
@@ -102,7 +102,9 @@ export default SlackFunction(def, async ({ inputs, client, env }) => {
         }
       } catch (e) {
         console.log(e);
-        modalMessage = e;
+        modalMessage =
+          "*:warning: Apologies! Failed to configure this app due to the following error:*\n>" +
+          e;
       }
       // nothing to return if you want to close this modal
       return buildModalUpdateResponse(modalMessage);

--- a/functions/internals/trigger_operations.ts
+++ b/functions/internals/trigger_operations.ts
@@ -122,7 +122,7 @@ async function joinChannel(
       console.log(`auth.test API result: ${JSON.stringify(authTest)}`);
     }
     const error =
-      `:warning: Failed to join <#${channelId}> due to "${response.error}" error. This workflow is unable to add <@${authTest.user_id}> to private channels and DMs. For those conversations, please invite the bot user in advance :bow:`;
+      `*:warning: Failed to join <#${channelId}> due to "${response.error}" error!*\n\nThis workflow is unable to add <@${authTest.user_id}> to private channels and DMs. For those conversations, please invite the bot user in advance :bow:`;
     console.log(error);
     return error;
   }

